documentation/requirements.txt

```diff
@@ -1,10 +1,11 @@
 # Python dependencies to build the documentation
 
 # != 3.5.0: https://github.com/sphinx-doc/sphinx/issues/8880
-Sphinx ~= 3.1, != 3.5.0
+Sphinx ~= 4.0, != 3.5.0
```

Comment:
You can remove the `!= 3.5.0` part, you're already restricting to at least 4.0 with the previous clause. (The comma is an AND here.)

Reply:
```suggestion
Sphinx ~= 4.0
```